### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by huit-tvachon**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/huit-tvachon/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 **Thanks for letting us show you around!**
 
+
```
Comment (huit-tvachon) [inline ReadMe.md:4]: !fix testing fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix not good

Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

File: ReadMe.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # ReviewNinja-Welcome
 
+
+
```
Comment (huit-tvachon) [inline ReadMe.md:4]: Remove extra white space !fix

File: ReadMe.md (hunk 1 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

File: ReadMe.md (hunk 2 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

File: ReadMe.md (hunk 1 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

File: ReadMe.md (hunk 2 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

Comment (huit-tvachon) [inline ReadMe.md:4]: !fix

Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

File: ReadMe.md (hunk 1 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !fixed

Comment (huit-tvachon) [inline ReadMe.md:4]: !resolve

File: ReadMe.md (hunk 2 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !resolve

File: ReadMe.md (hunk 1 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

File: ReadMe.md (hunk 2 above)
Comment (huit-tvachon) [inline ReadMe.md:4]: !resolved

**Pull request merged**